osbs/api.py

```diff
@@ -541,6 +542,16 @@ def get_token(self):
         return self.os.get_oauth_token()
 
     @osbsapi
+    def login(self, token):
+        token_file = utils.get_instance_token_file_name(self.os_conf.conf_section)
+        token_file_dir = os.path.dirname(token_file)
+        if not os.path.exists(token_file_dir):
+            os.makedirs(token_file_dir)
+        with open(token_file, 'w') as f:
+            os.fchmod(f.fileno(), 0o600)
```

Comment:
See e.g. http://stackoverflow.com/questions/5624359/write-file-with-specific-permissions-in-python